pkg/auth/oauth2/authentik.go

```diff
@@ -119,13 +133,18 @@ func (a *AuthentikAuthenticator) Callback() http.HandlerFunc {
 			http.Error(w, fmt.Sprintf("user info validation failed: %v", err), http.StatusUnauthorized)
 			return
 		}
-		rest.WriteDefaultHttpHeader(w, r.Context(), http.StatusOK)
-		_, err = w.Write(userData)
+
+		tokenDetails, err := loginAndGetJWT(userInfo)
 		if err != nil {
-			a.lc.Errorf("failed to write the response body: %v", err)
-			http.Error(w, fmt.Sprintf("fail to write the response body: %v", err), http.StatusInternalServerError)
+			a.lc.Errorf("failed to log in: %v", err)
+			http.Error(w, fmt.Sprintf("failed to log in: %v", err), http.StatusInternalServerError)
 			return
 		}
+
+		// Write the token to the response header and redirect to the redirect path
+		w.Header().Set("Access-Token", tokenDetails.AccessToken)
+		w.Header().Set("Refresh-Token", tokenDetails.RefreshToken)
```

Comment:
move the "Access-Token" and "Refresh-Token" headers to pkg/auth/jwt/constants.go, because the services might need to use them

Reply:
done